pkg/proto/configuration/blobstore/blobstore.proto

```diff
@@ -459,6 +459,27 @@ message LocalBlobAccessConfiguration {
     // intended to grow capacity beyond the size of memory.
     BlockDevice block_device = 10;
   }
+
+  message InMemoryDigestLocationMap {}
+
+  message FileBackedDigestLocationMap {
+    // Directory to house the digest-location map's state files.
+    string directory = 1;
```

Comment:
I am going to ask something from you that is technically speaking wrong, but it will make both our lives a bit easier.

I have a change on my side that I will likely push within the next couple of weeks that will change LocalBlobAccess to no longer require multiple digest-location maps for multiple instance names. This means that if we had a file-backed digest-location map, we wouldn't need to use a full directory. A single filename would suffice.

Could you already go ahead and write your code to work like this? Just use the same logic for the CAS and the AC. It's wrong in case you use multiple instances right now, but that will get fixed by me.